### DISCUSSION TITLE
with link to note's assets folder

### DISCUSSION THREAD

**Discussion by daeh** [SilentVoid13/Templater#485] [Templates Showcase]
I often want to open the folder containing all of the assets attached to a note. For me, these are located in in folder named `{note_basename}.assets` (e.g. `./_resources/{note_basename}.assets`). Here's a template that gives a clickable link to open that folder (on MacOS).

```md
---
created: <% tp.date.now('YYYY-MM-DD') %>T<% tp.date.now('HH-mm') %>
assets: "<% tp.file.path(false).split('/').slice(0, -1).join('/') + '/_resources/' + tp.file.title + '.assets/' %>"
tags: 
- LabNote 
---
# <% tp.file.title %>

_[assets](<% 'file:/' + encodeURIComponent(tp.file.path(false).split('/').slice(0, -1).join('/') + '/_resources/' + tp.file.title + '.assets/').replaceAll('%2F', '/') %>)_

---

<% tp.file.cursor(1) %>
```

gist here: https://gist.github.com/daeh/0f2ef3a63fff5d4fc5b7943f0e3f373d

**Comment by depressed-squidward** [comment]
That is fantastic! Thanks for this.
